src/Detector/ParserHelper.cs

```diff
@@ -19,8 +20,8 @@ namespace Microsoft.Oryx.Detector
     internal static class ParserHelper
     {
         /// <summary>
-        /// Parse a .toml file into a TomlTable from the Nett library.
-        /// See https://github.com/paiden/Nett for more information.
+        /// Parse a .toml file into a TomlTable from the Tomlyn library.
+        /// See https://github.com/xoofx/Tomlyn for more information.
```

Comment:
Is this just a new version of `Nett` tool?

Reply:
No, this is a new tool which has similar functions as `Nett`